build-system/server/new-server/transforms/utilities/cdn.ts

```diff
@@ -32,6 +32,11 @@ function getMinifiedName(minifiedBasename: string): string {
 
 /**
  * Convert an existing URL to one from the local `serve` command.
+ * @param url the url
+ * @param pathnames
+ * @param extension
+ * @param port
+ * @param useMaxNames
```

Comment:
Ah! I removed the requirement in the .eslintrc.
JSDoc can still be helpful to either describe parameters or as a function description via english prose.

Reply:
I definitely agree that it can, but for TS I'm not sure that we should require it/I don't think we want eslint-autofix to generate these empty `@param` annotations

Reply:
If this is no longer a JSDoc, it just a comment that will get out of date. Prefer to remove useless comments.